src/elements/content-uploader/__tests__/IconName.test.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { render, screen } from '../../../test-utils/testing-library';
+import IconName from '../IconName';
+import { STATUS_ERROR, STATUS_IN_PROGRESS } from '../../../constants';
+
+describe('elements/content-uploader/IconName', () => {
+    const renderComponent = (props = {}) => {
+        const defaultProps = {
+            extension: 'pdf',
+            name: 'test-item',
+            isResumableUploadsEnabled: false,
+            status: 'pending',
+        };
+        render(<IconName {...defaultProps} {...props} />);
+    };
+
+    test('renders component correctly', () => {
+        renderComponent();
+
+        expect(screen.getByRole('img', { name: 'File' })).toBeVisible();
```

Comment:
should we use `toBeVisible` over `toBeInTheDocument` if we aren't changing the visibility attributes for the element?